wd_core.au3

```diff
@@ -1710,6 +1712,7 @@ EndFunc   ;==>__WD_StripPath
 Func __WD_ConsoleWrite($sMsg, $iDebugLevel = Default, $iError = @error, $iExtended = @extended)
 	If $iDebugLevel = Default Or $_WD_DEBUG >= $iDebugLevel Then
 		If IsFunc($_WD_CONSOLE) Then
+			If IsNumber($iDebugLevel) And $iDebugLevel = $_WD_DEBUG_Full Then $sMsg = ':WD_DEBUG (' & $iError & '/' & $iExtended & ') : ' & $sMsg
```

Comment:
oops.... this should be in separate PR

Reply:
take a look on: [wd_core: reverting some code](https://github.com/Danp2/au3WebDriver/pull/285/commits/23118fc893e38fdaa2c894acd26169862d021aa4)